helm/holmes/templates/holmes.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-holmes
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: holmes
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: holmes
+  template:
+    metadata:
+      labels:
+        app: holmes
+    spec:
+      {{- if .Values.customServiceAccountName }}
+      serviceAccountName: {{ .Values.customServiceAccountName }}
+      {{ else }}
+      serviceAccountName: {{ .Release.Name }}-holmes-service-account
+      {{- end }}
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+      {{- toYaml .Values.imagePullSecrets | nindent 6 }}
+      {{- end }}
+      containers:
+      - name: holmes
+        image: "{{ .Values.registry }}/{{ .Values.image }}"
+        imagePullPolicy: Always
+        command: ["python3", "-u", "server.py"]
+        env:
+          - name: LOG_LEVEL
+            value: {{ .Values.logLevel }}
+          - name: ALLOWED_TOOLSETS
+            value: {{ .Values.allowedToolsets }}
+          {{ if .Values.certificate -}}
+          - name: CERTIFICATE
+            value: {{ .Values.certificate }}
+          {{- end }}
+          {{ if .Values.openaiKey -}}
+          - name: OPENAI_API_KEY
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.secretName }}
+                key: openaiKey
+          - name: OPENAI_ENDPOINT
+            value: {{ .Values.openaiEndpoint }}
+          - name: OPENAI_TYPE
+            value: {{ .Values.openaiType }}
+          - name: OPENAI_VERSION
+            value: {{ .Values.openaiVersion }}
+          - name: OPENAI_DEPLOYMENT
+            value: {{ .Values.openaiDeployment }}
+          - name: AI_MODEL
```

Comment:
I think the names you're using here don't all match up to the names read in Config.load_from_env. 
E.g. AI_MODEL should just be MODEL

Reply:
removing